Remotion/Core/Core/ServiceLocation/ImplementationForAttribute.cs

```diff
@@ -84,7 +83,7 @@ public Type ServiceType
     /// <para>
     /// When using the <see cref="ServiceLocator"/>, <see cref="ServiceLocation.RegistrationType.Single"/> and <see cref="ServiceLocation.RegistrationType.Compound"/>
     /// registrations must be resolved via <see cref="IServiceLocator.GetInstance(System.Type)"/> and <see cref="ServiceLocation.RegistrationType.Multiple"/> 
-    /// registrations must be resolved via <see cref="IServiceLocator.GetAllInstances(System.Type)"/>. Otherwise, an <see cref="ActivationException"/> is thrown.
+    /// registrations must be resolved via <see cref="IServiceLocator.GetAllInstances(System.Type)"/>. Otherwise, an <see cref="Remotion.ServiceLocation.ActivationException"/> is thrown.
```

Comment:
```suggestion
    /// registrations must be resolved via <see cref="IServiceLocator.GetAllInstances(System.Type)"/>. Otherwise, an <see cref="ActivationException"/> is thrown.
```

should be sufficient?